### ISSUE TITLE
Out of balance guild prices create Auction House arbitrage opportunities on server boot.

### ISSUE THREAD

**Issue by Deadwing888** [DarkstarProject/darkstar#2460]
The major culprits are:

Silk thread (Lower Jeuno Tenshodo): 
Retail Bid- 1,488 | Retail Ask- 77,280  | DSP bid: ~20k on server boot (4x spot price, 1.5M free gil)

Spider Web (Windy Woods/Selbina CC guild)
Retail bid- 990 | Retail ask- N/A | DSP bid: ~10k on server boot (2x spot price, 400k x2 free gil)

Velvet Cloth (Windy Woods/Selbina CC guild)
Retail bid- 710 | Retail ask- 26,796 | DSP bid: ~10k on server boot (2x spot price, 300k x2 free gil)

Blk. Tiger Fang (Windy Woods BC guild)
Retail bid- 642 | Retail ask- 12,693 | DSP bid: ~4k on server boot (2x spot price, 500k free gil)

Scorpion Claw (Windy Woods BC guild)
Retail Bid- 409 | Retail ask- N/A | DSP bid: ~2.5k on server boot (1.5x spot price, 150k free gil)

Scorpion Shell (Windy Woods BC guild)
Retail Bid- 632 | Retail ask- 12,166 | DSP bid: ~4k on server boot (1.3x spot price, 150k free gil)

Additionally the following guild items allow for straight money printing via pump and dump tactics:

Karakul Thread (Al Zabi)
Silk Thread (Windy woods+Selbina)
Tama-Hagane (Norg+Lower Jeuno)

These are the major problem areas, but I'm noticing the bid ask spread at guilds in retail is much wider than DSP across the board.

**Comment by teschnei** [comment]
I have no idea what the issue is supposed to be here

**Comment by Deadwing888** [comment]
tl;dr: The guilds are willing to buy certain items at a price that is too high.

**Comment by nasomi** [comment]
Retail buy/sell rates are independent, but dsp is a fixed rate. if a sell range is 60-100, and base buyback rate is 70%, you buy complete stock for 60*100=6000, price rockets to 100, buyback is now 70*100=7000, you make 1000 profit. Many items have overlap of this nature.

**Comment by teschnei** [comment]
what does this have to do with the auction house?

**Comment by Deadwing888** [comment]
That's where the free market price is discovered; the price against which the guild prices can be judged as too high.

On populated darkstar servers, after every maint or zone crash half a dozen people log on and buy 2M worth of stuff off the AH to sell to the guilds for ~5M. This time last year the servers leaked gil so badly that valuable items were bartered rather than sold. This issue is one of the remaining contributors to the funny money still sloshing around.

**Comment by TeoTwawki** [comment]
You are essential saying "My server can't handle a player run economy combined with retail features". Thats not a bug with darkstar.

**Comment by teschnei** [comment]
how does this work on retail?

**Comment by Deadwing888** [comment]
No, I'm saying your non-retail code can't run an XI economy properly.

I posted the specific retail bid and ask prices for the items in question as part of the issue.

**Comment by TeoTwawki** [comment]
Except you are utterly wrong. The fact you can buy low and sell high isn't non retail in the slightest. If you intend to say a guild items min/max are wrong, that still has nothing to do with the AH. Auction house prices on retail are set by players, and have jack to do with retails guild pricing. I was that jerk on retail exploiting the guilds in exactly the way you describe, easily 3mil every time maint ended if I was fast enough. Correcting an incorrect guild price is simple enough, but this looks like it is the same thing both of you talked to me about in PM on IRC.

**Comment by TeoTwawki** [comment]
Also, you didn't answer Kj's question.

**Comment by teschnei** [comment]
I still don't really understand what the issue is here - are the guilds paying too much? should they be starting stocked? what am I supposed to do about the auction house?

**Comment by R3P0FFXI** [comment]
The guild NPC's react as they are supposed to KJ they behave exactly as they do in retail. This is probably something that can be adjusted on the individual servers should they choose to adjust them. It sounds to me like they are upset that X number of items are stocked at Y price on server up or day change and its to many items for the amount of players on a given server.

**Comment by xdemolish** [comment]
is this even an issue for us, or just specific servers and their economy troubles?

**Comment by teschnei** [comment]
Only if it's not retail

**Comment by TeoTwawki** [comment]
I believe there was a genuine problem with the guilds price min/max, or the defaults pricing and stock, it was just explained really badly and made confusing by bringing the auction house into it (AH pricing has zero to do with anything, bringing AH into it only leads to confusion).

**Comment by nasomi** [comment]
It has nothing to do with AH, the issue is the buy back price. In dsp it is set to 30% of the sale price, but that is not how it's done in retail. Check any guild npc and buy an item, note it's price, then go to resell and note the price.

**Comment by teschnei** [comment]
Guide buyback is not static in either dsp or retail

**Comment by TeoTwawki** [comment]
> It has nothing to do with AH

yeah I already said that

**Comment by towbes** [comment]
One example where the min/max is a problem is line 63 in guild_shops.sql 
INSERT INTO `guild_shops` VALUES ('513', '701', '6615', '37044', '165', '33', '65');

The max price should never exceed 3x the min price in order to prevent players from buying out an item and then selling the next game day for profit.
